go.mod

```diff
@@ -5,21 +5,21 @@ go 1.18
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/fatih/structtag v1.1.0
-	github.com/go-logr/logr v1.2.2
+	github.com/go-logr/logr v1.2.3
 	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
 	github.com/iancoleman/strcase v0.2.0
 	github.com/kr/text v0.2.0
 	github.com/markbates/inflect v1.0.4
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.18.1
+	github.com/onsi/gomega v1.19.0
 	github.com/operator-framework/api v0.15.1-0.20220624132056-decf74800a17
 	github.com/operator-framework/helm-operator-plugins v0.0.12-0.20220616200420-1a695cb9f6a1
-	github.com/operator-framework/java-operator-plugins v0.6.0
+	github.com/operator-framework/java-operator-plugins v0.7.0
```

Comment:
I just noticed that other dependencies than the java-operator-plugins dependency are being updated as well. Is this because the other dependency updates are required as part of the java-operator-plugins bump?

If not, I think it may be best to make sure we are just bumping the java-operator-plugin and leave the rest of the bumps to a follow-up PR (one that is specifically for bumping SDK dependencies to support k8s 1.25).

Reply:
All other dependencies got updated once I did `go mod tidy`.